tests/unit/TechRecordsService.unitTest.ts

```diff
@@ -713,19 +718,86 @@ describe("updateTechRecord", () => {
       techRecord.vin = "XMGDE02FS0H012345";
       techRecord.primaryVrm = "JY58FPP";
       try {
-        expect(await techRecordsService.updateTechRecord(techRecord, msUserDetails)).toThrowError();
+        expect(await techRecordsService.updateTechRecord(techRecord, msUserDetails, newStatusCode)).toThrowError();
       } catch (errorResponse) {
         expect(errorResponse).toBeInstanceOf(HTTPError);
         expect(errorResponse.statusCode).toEqual(404);
         expect(errorResponse.body).toEqual(HTTPRESPONSE.RESOURCE_NOT_FOUND);
       }
     });
   });
+
+  context("A technical record status is updated to archived via PUT verb", () => {
+    it("should change the records status to archived and set audit details and create no other record", async () => {
+    const newStatusCode = STATUS.ARCHIVED;
+    const techRecord: any = cloneDeep(records[43]);
+    console.log("New test \n", techRecord.techRecord[0].statusCode);
+    const MockDAO = jest.fn().mockImplementation(() => {
+      return {
+        getBySearchTerm: () => {
+          return Promise.resolve({
+            Items: [cloneDeep(records[43])],
+            Count: 1,
+            ScannedCount: 1
+          });
+        }
+      };
+    });
+    const techRecordsService = new TechRecordsService(new MockDAO());
+    const updatedTechRec: any = await techRecordsService.createAndArchiveTechRecord(techRecord, msUserDetails, newStatusCode);
+    expect(STATUS.ARCHIVED).toEqual(updatedTechRec.techRecord[0].statusCode);
```

Comment:
expect(updatedTechRec.techRecord[0].statusCode).toEqual(STATUS.ARCHIVED); it makes more sense this way